test/input_test.py

```diff
@@ -1,28 +1,35 @@
 from unittest import TestCase
-from passflip.input import create_prompt_message
+import passflip.input
 
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 class InputPromptMessageTest(TestCase):
 
-    def test_that_colon_is_added_to_message(self):
-        self.given_the_message("Foo")
-        self.when_converted_to_a_prompt_message()
-        self.then_the_result_will_be("Foo: ")
+    def setUp(self):
+        self.validate = False;
 
-    def test_that_again_is_added_to_message(self):
-        self.given_the_message("Foo")
-        self.when_converted_to_a_prompt_message_with_again()
-        self.then_the_result_will_be("Foo again: ")
+    def test_that_user_can_input_password(self):
+        expected = "foocar"
+        self.when_user_is_prompted_for_password_and_they_input(expected)
```

Comment:
These tests are fine, but they're a little confusing.

What's "the result" meant to be from prompting a user for a password?